types/es-abstract/helpers/assertRecord.d.ts

```diff
@@ -11,6 +11,8 @@ declare namespace assertRecord {
     interface Predicates {
         'Property Descriptor': (
             ES: {
+                // This is necessary to avoid "TS2345: Object literal may only specify known properties":
+                [method: string]: any;
```

Comment:
```suggestion
                [method: string]: unknown;
```

Reply:
Doing that will cause “TS2345: Index signature is missing in type 'ESAbstract'”.

Reply:
hmm - maybe instead this object should be something like `Pick<ES, 'Type'>`?